src/adapters/convex/products/staking/metadata/ethereum.metadata.json

```diff
@@ -1,4 +1,28 @@
 {
+  "0x4C204392a9E4d5f8390541971B59FC95B24782b5": {
+    "protocolToken": {
+      "address": "0x4C204392a9E4d5f8390541971B59FC95B24782b5",
+      "name": "pUSD-3CRV-A1000 Convex Deposit",
+      "symbol": "cvxpUSD3crv",
+      "decimals": 18
+    },
+    "underlyingToken": {
+      "address": "0x2482DFb5A65D901d137742AB1095f26374509352",
+      "name": "pUSD-3CRV-A1000",
+      "symbol": "pUSD3crv",
+      "decimals": 18
+    },
+    "rewardTokens": [
+      {
+        "address": "0xD533a949740bb3306d119CC777fa900bA034cd52",
+        "name": "Curve DAO Token",
+        "symbol": "CRV",
+        "decimals": 18
+      }
+    ],
+    "extraRewardTokens": [],
+    "extraRewardTokenManagers": []
```

Comment:
nit: If `extraRewardTokens` and `extraRewardTokenManagers` are supposed to be the same size, it'd have been better if they were the same array (in which each item has two fields, the token metadata and the token manager).

Otherwise we'd have to use `!` in typescript for array access or do an `undefined` check.

In any case, not worth changing it.